src/partials/footer-content.hbs

```diff
@@ -31,14 +31,13 @@
           <ul id="menu-interact" class="menu">
             <li class="menu-mailing-list"><a href="https://talk.owncloud.com" class="oc-footer-nav-link">Rocket.Chat</a></li>
             <li class="menu-forums"><a href="https://central.owncloud.org" class="oc-footer-nav-link">Forums</a></li>
-            <li class="menu-bug-tracker"><a href="https://owncloud.org/community" class="oc-footer-nav-link">Get involved</a></li>
-            <li class="menu-promote"><a href="https://owncloud.org/promote/" class="oc-footer-nav-link">Discuss ownCloud with others</a></li>
+            <li class="menu-promote"><a href="https://owncloud.com/promote/" class="oc-footer-nav-link">Discuss ownCloud with others</a></li>
```

Comment:
This link is more or less dead/useless and can be removed IMO as we have a `Contribute` link already

Reply:
fixed